packages/juce-blueprint/src/index.tsx

```diff
@@ -0,0 +1,45 @@
+import BlueprintBackend from './lib/BlueprintBackend';
+import BlueprintRenderer, { BlueprintTracedRenderer } from './lib/BlueprintRenderer';
+
+export { default as NativeMethods } from './lib/NativeMethods';
+export { default as EventBridge } from './lib/EventBridge';
+
+export * from './components/View';
+export * from './components/ScrollView';
+export * from './components/Canvas';
+export * from './components/Text';
+export * from './components/Image';
+
+let __renderStarted = false;
```

Comment:
We could move the below out into `Blueprint.ts` and have `index.tsx` literally just be a barrel file but I think that would require users to use a named import for `Blueprint` in things like `index.js` etc.

Reply:
Yea, this is cool with me